native/src/routes/Pois.tsx

```diff
@@ -110,15 +107,15 @@ const Pois = ({ pois, language, cityModel, route, navigation }: PoisProps): Reac
   // Wait for followUserLocation change before moving the camera to avoid position lock
   // https://github.com/rnmapbox/maps/issues/1079
   useEffect(() => {
-    if (!followUserLocation && selectedFeature && cameraRef.current && prevUrlSlug !== urlSlug) {
```

Comment:
Why not needed anymore?

Reply:
this prevented the `flyTo` for internal linking. I somehow can not find an issue by removing that. Did you find anything?